python/intro/README.md

```diff
@@ -0,0 +1,405 @@
+```
+
+Functions are grouped into applications so by specifying `--app pythonapp`
+we're implicitly creating the application `pythonapp` and associating our
+function with it.
+
+Specifying `--local` does the deployment to the local server but does
+not push the function image to a Docker registry--which would be necessary if
+we were deploying to a remote Fn server.
+
+The output message
+`Updating route /pythonfn using image fndemouser/pythonfn:0.0.2...`
+lets us know that the function packaged in the image
+`fndemouser/pythonfn:0.0.2` has been bound by the Fn server to the route
+`/pythonfn`.  We'll see how to use the route below.
+
+Note that the containing folder name 'pythonfn' was used as the name of the
+generated Docker container and used as the name of the route that
+container was bound to.
+
+The fn CLI provides a couple of commands to let us see what we've deployed.
+`fn apps list` returns a list of all of the defined applications.
+
+![user input](images/userinput.png)
+>```sh
+> fn apps list
+>```
+
+Which, in our case, returns the name of the application we created when we
+deployed our pythonfn function:
+
+```sh
+pythonapp
+```
+
+We can also see the functions that are defined by an application.  Since
+functions are exposed via routes, the `fn routes list <appname>` command
+is used.  To list the functions included in `pythonapp` we can type:
+
+![user input](images/userinput.png)
+>```sh
+> fn routes list pythonapp
+>```
+
+```sh
+path		image				endpoint
+/pythonfn	fndemouser/pythonfn:0.0.2	localhost:8080/r/pythonapp/pythonfn
+```
+
+The output confirms that pythonapp contains a `pythonfn` function that is
+implemented by the Docker container `fndemouser/pythonfn:0.0.2` which may be
+invoked via the specified URL.  Now that we've confirmed deployment was
+successful, let's call our function.
+
+## Calling Your Deployed Function
+
+There are two ways to call your deployed function.  The first is using
+the `fn` CLI which makes invoking your function relatively easy.  Type
+the following:
+
+![user input](images/userinput.png)
+>```sh
+> fn call pythonapp /pythonfn
+>```
+
+which results in our familiar output message.
+
+```json
+{"message":"Hello World"}
+```
+
+Of course this is unchanged from when you ran the function locally.
+However when you called `pythonapp /pythonfn` the fn server looked up the
+`pythonapp` application and then looked for the Docker container image
+bound to the `/pythonfn` route.
+
+The other way to call your function is via HTTP.  The Fn server exposes our
+deployed function at `http://localhost:8080/r/pythonapp/pythonfn`, a URL that
+incorporates our application and function route as path elements.
+
+Use curl to invoke the function:
+
+![user input](images/userinput.png)
+>```sh
+> curl -H "Content-Type: application/json" http://localhost:8080/r/pythonapp/pythonfn
+>```
+
+The result is once again the same.
+
+```js
+{"message":"Hello World"}
```

Comment:
js -> json

Reply:
Tested and they look the same. Changed to json for accuracy.